This patch reduces the priority of the posix_cpu-timers kthread
from 99 to 98. to make room for another high-prio RT thread.

Signed-off-by: Remy Bohmer <linux@bohmer.net>
---
 kernel/posix-cpu-timers.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-2.6.23/kernel/posix-cpu-timers.c
===================================================================
--- linux-2.6.23.orig/kernel/posix-cpu-timers.c	2008-01-14 12:55:39.000000000 +0100
+++ linux-2.6.23/kernel/posix-cpu-timers.c	2008-01-14 12:55:41.000000000 +0100
@@ -1479,7 +1479,7 @@ static int posix_cpu_thread_call(struct 
 		p->flags |= PF_NOFREEZE;
 		kthread_bind(p, cpu);
 		/* Must be high prio to avoid getting starved */
-		param.sched_priority = MAX_RT_PRIO-1;
+		param.sched_priority = MAX_RT_PRIO-2;
 		sched_setscheduler(p, SCHED_FIFO, &param);
 		per_cpu(posix_timer_task,cpu) = p;
 		break;
